TOP_DIR=${PWD}
: ${MOTHUR_REF_DATA:=$TOP_DIR}
if [ ! -d "$MOTHUR_REF_DATA" ] ; then
echo Missing stability.files in the directory $MOTHUR_REF_DATA
exit 1
fi
$MOTHUR_REF_DATA/mothur "#make.contigs(file=stability.files, processors=10)"
rm *.fastq
$MOTHUR_REF_DATA/mothur "#summary.seqs(fasta=stability.trim.contigs.fasta, processors=10); screen.seqs(fasta=stability.trim.contigs.fasta, group=stability.contigs.groups, maxambig=0, maxlength=295); summary.seqs(fasta=stability.trim.contigs.good.fasta); unique.seqs(fasta=stability.trim.contigs.good.fasta); count.seqs(name=stability.trim.contigs.good.names, group=stability.contigs.good.groups); pcr.seqs(fasta=$MOTHUR_REF_DATA/silva.nr_v119.align, start=13862, end=23444, keepdots=F, processors=10); system(mv $MOTHUR_REF_DATA/silva.nr_v119.pcr.align silva.v4.fasta); align.seqs(fasta=stability.trim.contigs.good.unique.fasta, reference=silva.v4.fasta); summary.seqs(fasta=stability.trim.contigs.good.unique.align, count=stability.trim.contigs.good.count_table); screen.seqs(fasta=stability.trim.contigs.good.unique.align, count=stability.trim.contigs.good.count_table, summary=stability.trim.contigs.good.unique.summary, start=8, end=9582, maxhomop=8); summary.seqs(fasta=stability.trim.contigs.good.unique.good.align); filter.seqs(fasta=stability.trim.contigs.good.unique.good.align, vertical=T, trump=.); unique.seqs(fasta=stability.trim.contigs.good.unique.good.filter.fasta, count=stability.trim.contigs.good.good.count_table);pre.cluster(fasta=stability.trim.contigs.good.unique.good.filter.unique.fasta, count=stability.trim.contigs.good.unique.good.filter.count_table, diffs=2); chimera.uchime(fasta=stability.trim.contigs.good.unique.good.filter.unique.precluster.fasta, count=stability.trim.contigs.good.unique.good.filter.unique.precluster.count_table, dereplicate=t);remove.seqs(fasta=stability.trim.contigs.good.unique.good.filter.unique.precluster.fasta, accnos=stability.trim.contigs.good.unique.good.filter.unique.precluster.uchime.accnos);classify.seqs(fasta=stability.trim.contigs.good.unique.good.filter.unique.precluster.pick.fasta, count=stability.trim.contigs.good.unique.good.filter.unique.precluster.uchime.pick.count_table, reference=$MOTHUR_REF_DATA/silva.nr_v119.align, taxonomy=$MOTHUR_REF_DATA/silva.nr_v119.tax, cutoff=80); remove.lineage(fasta=stability.trim.contigs.good.unique.good.filter.unique.precluster.pick.fasta, count=stability.trim.contigs.good.unique.good.filter.unique.precluster.uchime.pick.count_table, taxonomy=stability.trim.contigs.good.unique.good.filter.unique.precluster.pick.nr_v119.wang.taxonomy, taxon=Chloroplast-Mitochondria-unknown-Archaea-Eukaryota); summary.seqs(fasta=stability.trim.contigs.good.unique.good.filter.unique.precluster.pick.pick.fasta, processors=10)"
rm silva*
rm *.align
$MOTHUR_REF_DATA/mothur "#dist.seqs(fasta=stability.trim.contigs.good.unique.good.filter.unique.precluster.pick.pick.fasta, cutoff=0.20, processors=10, output=lt); cluster(phylip=stability.trim.contigs.good.unique.good.filter.unique.precluster.pick.pick.phylip.dist, count=stability.trim.contigs.good.unique.good.filter.unique.precluster.uchime.pick.pick.count_table, cutoff=0.05, precision=1000, method=furthest); make.shared(list=stability.trim.contigs.good.unique.good.filter.unique.precluster.pick.pick.phylip.fn.unique_list.list, count=stability.trim.contigs.good.unique.good.filter.unique.precluster.uchime.pick.pick.count_table); classify.otu(list=stability.trim.contigs.good.unique.good.filter.unique.precluster.pick.pick.phylip.fn.unique_list.list, count=stability.trim.contigs.good.unique.good.filter.unique.precluster.uchime.pick.pick.count_table, taxonomy=stability.trim.contigs.good.unique.good.filter.unique.precluster.pick.nr_v119.wang.pick.taxonomy); make.biom(shared=stability.trim.contigs.good.unique.good.filter.unique.precluster.pick.pick.phylip.fn.unique_list.shared, constaxonomy=stability.trim.contigs.good.unique.good.filter.unique.precluster.pick.pick.phylip.fn.unique_list.unique.cons.taxonomy)"
